cmd/zrepl/zrepl.c

```diff
@@ -947,15 +1063,34 @@ uzfs_zrepl_walk_pool_directory(void)
 int
 main(int argc, char **argv)
 {
-	int 		rc;
-	kthread_t	*conn_accpt_thrd;
-	kthread_t	*uzfs_mgmt_thread;
-	char		*target_addr = NULL;
+	int	rc;
+	int	i = 0;
+	const char	*cmd_name = NULL;
+
+	if (argc < 2) {
+		help();
+		return (1);
+	}
+
+	cmd_name = argv[1];
+
+	if ((rc = find_command(cmd_name, &i)) != 0) {
+		help();
+		return (1);
+	}
 
 	pthread_t slf = pthread_self();
 	snprintf(tinfo, sizeof (tinfo), "m#%d.%d",
 	    (int)(((uint64_t *)slf)[0]), getpid());
 
+	if (getenv("CONFIG_LOAD_ENABLE") == NULL) {
+		printf("enabled auto import (reading of zpool.cache)\n");
+		zfs_autoimport_disable = 0;
+	} else {
+		printf("not reading zpool.cache files\n");
```

Comment:
so zfs_autoimport_disable is set to true by default? weird. I would expect we need to explicitly set it to 1 here.